client/cz-component-dash.html

```diff
@@ -100,7 +100,7 @@
       is: "cz-component-dash",
 
       properties: {
-        users: {
+        userConfigs: {
```

Comment:
I can anticipate disapproval for this rename, it's has a weaker line of reasoning than `searchQuery.query`.